packages/entities/entities-plugins/src/components/free-form/shared/composables.ts

```diff
@@ -469,6 +469,18 @@ export function useField<T = unknown>(name: MaybeRefOrGetter<string>) {
   }
 }
 
+export function useFormData<T>(name: MaybeRefOrGetter<string>) {
+  const { formData } = useFormShared()
+  const fieldPath = useFieldPath(name)
+  const value = computed<T>({
+    get: () => get(formData, utils.toArray(fieldPath.value)),
+    set: (v) => set(formData, utils.toArray(fieldPath.value), v),
+  })
+  return {
+    value,
+  }
+}
+
```

Comment:
Please also use this function to refactor the [`value` in `useField`](https://github.com/Kong/public-ui-components/blob/b92961d5dfaa18511c80c82d412d70b44c46b33d/packages/entities/entities-plugins/src/components/free-form/shared/composables.ts#L449)